content/en/docs/Configuration/custom-dashboard.md

```diff
@@ -0,0 +1,250 @@
+    * *min* and *max*: domain for Y-values. When unset, charts implementations should usually automatically adapt the domain with the displayed data.
+    * *xAxis*: type of the X-axis, can be one of _time_ (default) or _series_. When set to _series_, only one datapoint per series will be displayed, and the chart type then defaults to _bar_.
+    * *aggregator*: defines how the time-series are aggregated when several are returned for a given metric and label set. For example, if a Deployment creates a ReplicaSet of several Pods, you will have at least one time-series per Pod. Since Kiali shows the dashboards at the workload (ReplicaSet) level or at the application level, they will have to be aggregated. This field can be used to fix the aggregator, with values such as _sum_ or _avg_ (full list available [in Prometheus documentation](https://prometheus.io/docs/prometheus/latest/querying/operators/#aggregation-operators)). However, if omitted the aggregator will default to _sum_ and can be changed from the dashboard UI.
+    * *aggregations*: list of labels eligible for aggregations / groupings (they will be displayed in Kiali through a dropdown list)
+      * *label*: Prometheus label name
+      * *displayName*: name to display in Kiali
+      * *singleSelection*: boolean flag to switch between single-selection and multi-selection modes on the values of this label. Defaults to _false_.
+    * *groupLabels*: a list of Prometheus labels to be used for grouping. Similar to *aggregations*, except this grouping will be always turned on.
+    * *sortLabel*: Prometheus label to be used for the metrics display order.
+    * *sortLabelParseAs*: set to _int_ if *sortLabel* needs to be parsed and compared as an integer instead of string.
+  * *include*: to include another dashboard, or a specific chart from another dashboard. Typically used to compose with generic dashboards such as the ones about _MicroProfile Metrics_ or _Micrometer_-based JVM metrics. To reference a full dashboard, set the name of that dashboard. To reference a specific chart of another dashboard, set the name of the dashboard followed by `$` and the name of the chart (ex: `include: "microprofile-1.1$Thread count"`).
+* *externalLinks*: a list of related external links (e.g. to Grafana dashboards)
+  * *name*: name of the related dashboard in the external system (e.g. name of a Grafana dashboard)
+  * *type*: link type, currently only _grafana_ is allowed
+  * *variables*: a set of variables that can be injected in the URL. For instance, with something like _namespace: var-namespace_ and _app: var-app_, an URL to a Grafana dashboard that manages _namespace_ and _app_ variables would look like:
+_\http://grafana-server:3000/d/xyz/my-grafana-dashboard?var-namespace=some-namespace&var-app=some-app_. The available variables in this context are *namespace*, *app* and *version*.
```

Comment:
The backslash in the grafana url seems to be specific for adoc. It was to escape the url and prevent asciidoc to render it as a link. But markdown is rendering the backslash. Probably it is not needed.